api/TODO.md

```diff
@@ -0,0 +1,11 @@
+# TODO
+
+These tasks must be completed before the API package can be released from this repository.
+
+- [x] remove files specific to the API git repo
+- [ ] add to lerna monorepo
+  - [ ] add tsconfig files
+  - [ ] add web testing
```

Comment:
Should we the API to the webworker tests too? :thinking:

Reply:
Yeah I just consider that part of web testing. Have it in the draft for the follow up on this.

Reply:
Awesome thx :slightly_smiling_face: